.openshift/rocket-chat-ephemeral.json

```diff
@@ -262,6 +262,30 @@
                                         "mountPath": "/app/uploads"
                                     }
                                 ],
+                                "readinessProbe": {
+                                    "httpGet": {
+                                        "path": "/api/v1/info",
+                                        "port": 3000,
+                                        "scheme": "HTTP"
+                                    },
+                                    "initialDelaySeconds": 5,
+                                    "timeoutSeconds": 1,
+                                    "periodSeconds": 10,
+                                    "successThreshold": 1,
+                                    "failureThreshold": 3
```

Comment:
@geekgonecrazy I didn't know it is 'normal' for RocketChat to take many seconds (minutes??) to start on larger servers. If this is indeed the norm, I agree that the numbers I submitted in my PR are too low (they are the kubernetes defaults).

The readiness probe is the one that checks if the pod is ready on startup. It is common to see a larger delay on this one because of situations like the one you describe. That would be the one we would increase. The liveness probe ensures the app passes a check every X seconds to ensure it hasn't transited into a bad state.

I assume that on small-medium sized instances, the startup time is relatively short. I'm not sure if 300 seconds is a sane default for most people. Then if you manage a very large instance on kubernetes I would assume that you know you can increase these numbers. How about a delay of 15 seconds? Would that be suitable for most people?